### DISCUSSION TITLE
How to add .qmd file to github issue?

### DISCUSSION THREAD

**Discussion by victorfeagins** [quarto-dev/quarto-cli#2535] [Q&A] [answered]
From the github issues:
Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.

I got that the file type was not supported so I was curious how to include a .qmd file. Would it be though sharing the file though a link?

**Comment by cscheid** [comment, marked as answer]
You can either share a link or you can paste the entire contents of the file into the issue.

To ensure that the contents are escaped correctly, all you have to do is use a larger number of backticks than what your document uses. If you only use three ticks in your document, then you can surround it by four ticks:

````
```{r}
c(1,2,3)
```
````

If you happen to use four ticks, then five ticks also work:

`````
````
```{r}
c("it's", "ticks", "all", "the", "way", "down")
```
````
`````

**Reply by victorfeagins** [reply]
Cool that is what I thought.
